lib/data/firebase_const.dart

```diff
@@ -19,6 +19,8 @@ const imageSizes = "sizes";
 const imagePath = "path";
 const imageFile = "file";
 
+const zero = 0;
```

Comment:
can we safely assume that 0 will remain 0? I'm not sure I understand why this is being made a const?

Reply:
As this seems to be only used to initialise `relatedLimit` to `0` in `lib/data/managers/firestore_manager.dart` I would have to agree with @raubreysmith; to be clear: I support avoiding the use of [Magic Numbers](https://en.wikipedia.org/wiki/Magic_number_(programming)) however `0` and sometimes `1` are universal constants which makes it uncommon to need to extract these to constants.

Reply:
Thanks @RichardSlater . I let it as 0